Iterable-iOS-SDK/IterableAppIntegration.m

```diff
@@ -104,4 +114,11 @@ + (void)userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNoti
         completionHandler();
 }
 
++ (IterableAction *)legacyDefaultActionFromPayload:(NSDictionary *)userInfo {
+    if (userInfo[ITBL_PAYLOAD_DEEP_LINK_URL] != nil)
```

Comment:
Not needed for approval. I think you missed open close braces for all single line if/else.  I see those in other parts of the code.